src/blockchain_db/lmdb/db_lmdb.cpp

```diff
@@ -351,98 +283,177 @@ typedef struct outtx {
     uint64_t local_index;
 } outtx;
 
+const std::string& mdb_databases::name(source database)
+{
+  static std::array<std::string, 18> names{
```

Comment:
Do these need to be `std::string`? This could be `const char*` with the entire array marked as `constexpr`. The return value would be `const char*` too, I'm not seeing `std::string` required usage anywhere. Its looks like its being used either as a C-string to LMDB or an error temp `std::string`.

Reply:
It's not really necessary indeed. I did it this way mainly because it's only initialized _once_ in any case (so there shouldn't be a measurable run-time overhead. I'll change the array type to a `const char*`.